java/client/test/org/openqa/selenium/ImplicitWaitTest.java

```diff
@@ -154,4 +155,28 @@ public void testShouldImplicitlyWaitForAnElementToBeVisibleBeforeInteracting() {
       fail("Element should have been visible");
     }
   }
+
+
+  @Test
+  @JavascriptEnabled
+  @Ignore({IE, PHANTOMJS, SAFARI, MARIONETTE})
+  public void testShouldRetainImplicitlyWaitFromTheReturnedWebDriverOfFrameSwitchTo() {
+    driver.manage().timeouts().implicitlyWait(3, SECONDS);
```

Comment:
lukeis,
That is fine.  We did 3 seconds to make sure there was actually wait after the findElements failed.  I will update to 1 second